index.bs

```diff
@@ -826,7 +826,7 @@ noticable asymmetry between the two branches. [[!HTML]]
   1. Set _pull_.[[branch1]] to _branch1Stream_.[[readableStreamController]].
   1. Set _pull_.[[branch2]] to _branch2Stream_.[[readableStreamController]].
   1. <a>Upon rejection</a> of _reader_.[[closedPromise]] with reason _r_,
-    1. If _teeState_.[[closedOrErrored]] is *true*, return *undefined*.
+    1. If _teeState_.[[closedOrErrored]] is *true*, return.
```

Comment:
I feel that this one should be return *undefined*, since ReadableStreamTee does sometimes return a defined value.